cove_ofds/templatetags/ofds_cove_tags.py

```diff
@@ -0,0 +1,11 @@
+from django import template
+
+register = template.Library()
+
+
+@register.filter(name="error_instance_display")
+def error_instance_display(input):
+    if isinstance(input, str):
+        return '"' + input + '"'
```

Comment:
Would this be neater with an f-string?

Reply:
As we haven't discussed which one we prefer as a group, I'm going to say it's ok to pass for now. Also, I'm pretty sure that error_instance_display func will need to be expanded in the future so we can come back to it.